framework/src/araui/include/ui_widget_internal.h

```diff
@@ -65,6 +71,11 @@ typedef struct {
 	uint32_t d;
 } tween_info_t;
 
+typedef struct {
+	float u;
+	float v;
+} uv_t;
```

Comment:
This `uv_t` should be `ui_uv_t`, as you use below.

Reply:
You're right. I'll fix.